### PR TITLE
New package: helden-software-5.5.3

### PR DESCRIPTION
**Pull request by jcgruenhage**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

### CODE REVIEW

File: srcpkgs/helden-software/patches/0002-fix-file-locations.patch [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+diff --git a/usr/games/helden-software b/usr/games/helden-software
+index 9105ab5..e3622cd 100755
+--- a/usr/games/helden-software
++++ b/usr/games/helden-software
+@@ -2,6 +2,6 @@
+ JavaArgs="-Xms40m -Xmx256m"
+ HeldenArgs=""
+ if [ -f /etc/default/helden-software ] ; then
+-    . /etc/default/helden-software
++    source /etc/default/helden-software
```
Comment (abenson) [inline srcpkgs/helden-software/patches/0002-fix-file-locations.patch:16]: Why this change?

Comment (jcgruenhage) [reply srcpkgs/helden-software/patches/0002-fix-file-locations.patch:16]: Good question, I don't remember :D

I should probably drop that change there

Comment (jcgruenhage) [reply srcpkgs/helden-software/patches/0002-fix-file-locations.patch]: Resolved, I've removed that part now.

**Comment by jcgruenhage** [comment]
No, not stale, just needs a review and a merge.

File: srcpkgs/helden-software/patches/0001-remove-hsDebianMode-arg.patch [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+-HeldenArgs=-hsDebianMode
++HeldenArgs=""
+diff --git a/usr/games/helden-software b/usr/games/helden-software
+index 49d866b..9105ab5 100755
+--- a/usr/games/helden-software
++++ b/usr/games/helden-software
+@@ -1,6 +1,6 @@
+ #!/bin/sh
+ JavaArgs="-Xms40m -Xmx256m"
+-HeldenArgs=-hsDebianMode
```
Comment (Johnnynator) [inline srcpkgs/helden-software/patches/0001-remove-hsDebianMode-arg.patch:27]: Is this actually what you want in a system-wide installation. It looks to me, that this has two modes, "Debian" for a system installation and "local" if it can write next to its jar. Since currently launching it will give this warning:
image

Comment (jcgruenhage) [reply srcpkgs/helden-software/patches/0001-remove-hsDebianMode-arg.patch:27]: So that's what the Debian mode does, good to know. I couldn't find documentation for that parameter, so I turned it off. 

```text
void-packages on  helden-software-5.5.3_1 [$?⇕] took 13s 
❯ java -jar ~/Downloads/helden-deb/usr/lib/heldensoftware/helden5.jar --help
Der Parameter --help ist unbekannt!
Mit "/?" koennen alle moeglichen Parameter angezeigt werden.
Jun 13, 2022 2:13:05 PM helden.gui.ooOO.i:-1 o00000
WARNING: Ladezeit: 0.277081554

void-packages on  helden-software-5.5.3_1 [$?⇕] 
❯ java -jar ~/Downloads/helden-deb/usr/lib/heldensoftware/helden5.jar '/?'
---Parameterliste---
"-hsLocal"  setzt alle Pfade relative zum Jarfile, dieser Modus ist zB für USB-Sticks gedacht
"-hsLocalOff" schaltet den Localmodus wieder aus
"-ep" Durch diesen Parameter wird die Einstellungsdatei aus diesem Verzeichnis geladen.  z.B.  "-epd:\helden"
```

^ this doesn't list `-hsDebianMode` as a valid parameter anyway

In that case though, I'll remove that patch.
